### DISCUSSION TITLE
Binaries for other platforms in GitHub releases

### DISCUSSION THREAD

**Discussion by srfsh** [HemantKArya/BloomeeTunes#30] [General]
It would be highly appreciated for anyone trying out the project for the first time to grab a release from GitHub. I believe SourceForge has a bit unfriendly interface. Feel free to ignore, though.

**Comment by srfsh** [comment]
Oh, looks like the project has *some* Linux binaries. I wasn't able to see that on the latest stable release, though. It is only available in a prerelease.

**Comment by HemantKArya** [comment]
@srfsh Thanx for helping others, yes there are Linux binaries in pre-release section and Sourceforge, cause Sourceforge automatically imports latest release/pre-release from GitHub release. So, it is synced with both release and pre-release.

**Comment by srfsh** [comment]
> @srfsh Thanx for helping others, yes there are Linux binaries in pre-release section and Sourceforge, cause Sourceforge automatically imports latest release/pre-release from GitHub release. So, it is synced with both release and pre-release.

Oh, I forgot to mention but the pre-release binary does not work when trying to look up a song or trying to click one of the songs on the main screen and play it. How can I supply more information?

**Comment by HemantKArya** [comment]
Can you provide more details like which version of OS or type of OS you are using. And is stable release is working fine? Cause in mine Windows/Android/Linux (VMBox) are working fine.

**Reply by srfsh** [reply]
I am using the latest version of Artix Linux. As there were no latest release (BloomeeTunes-2.9.5-130) for Linux, I tried to ran in using flutter directly in the downloaded zip archive (after decompressing, obviously). The initial attempt of mine has failed with an error stating that `asset/.env` wasn't available. Because of that, the app refused to initialize. I created an empty file at that particular location. Then ran `flutter run -d linux` again. It worked this time, but I am experiencing the same error when trying to perform the aforementioned actions. They all fail with network connection error.

I am attacking the logs that I gathered as well. Hope it helps.

[log.txt](https://github.com/user-attachments/files/16715485/log.txt)

**Reply by HemantKArya** [reply]
Btw did you installed  libmpv/mpv on your system?

**Reply by srfsh** [reply]
Yup. I am using what is bundled with version `v0.38.0-dirty`, built on `Jul  3 2024 12:50:20`.

**Reply by HemantKArya** [reply]
Ahh, I guess may be 2 causes are behind this -
- it is not able to recognize `libmpv or mpv` in paths where the library` media_kit `is expecting.
- Another is the .env file is just empty (less likely but wait for other commits i will completely remove this cause right now I have no intention to keep `dotenv`).

also, when I was building Linux binaries in fedora Linux it was also producing error related to `mediakit` since it was not able to find `mpv` in path, I don't know why because it is working fine in `ubuntu/xubuntu` with same configuration but we still have to work on this problem more so that the `libmpv/mpv` will also bundled with Linux binaries and no one have to install `mpv` explicitly.

**Reply by srfsh** [reply]
In the mean time, is there anything I can do to help?
